packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy.types.ts

```diff
@@ -91,7 +91,9 @@ export type TSelectsFieldNames =
     | 'quick-strategy__type-strategy'
     | 'quick-strategy__symbol'
     | 'quick-strategy__trade-type'
-    | 'quick-strategy__duration-unit';
+    | 'quick-strategy__duration-unit'
+    | 'quick-strategy__duration-unit'
```

Comment:
@vinu-deriv can we remove duplicate types, I believe this got added accidentally while resolving conflicts or merging.